cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -282,7 +282,7 @@
 
   <property>
     <name>twill.security.worker.mount.secret</name>
-    <value>true</value>
+    <value>false</value>
```

Comment:
Given that we'll have secret mount in sidecar container, is there any scenario in which we want secret mount in worker runnable?

Reply:
No. Secret shouldn't be exposed to the environment where untrusted code runs